schema_based_on_diagram.sql

```diff
@@ -0,0 +1,46 @@
+CREATE TABLE patients(
+	id INT GENERATED BY DEFAULT AS IDENTITY PRIMARY KEY,
+	name VARCHAR(255),
+	date_of_birth DATE,
+);
+
+CREATE TABLE medical_histories(
+	id INT GENERATED BY DEFAULT AS IDENTITY PRIMARY KEY,
+	admitted_at TIMESTAMP,
+	patient_id INT REFERENCES patients(id),
+	status VARCHAR(255)
+);
+
+CREATE TABLE treatments(
+	id INT GENERATED BY DEFAULT AS IDENTITY PRIMARY KEY,
+	type VARCHAR(250),
+	name VARCHAR(250)
+);
+
+CREATE TABLE medical_histories_and_treatments(
+	treatment_id INT REFERENCES treatments(id),
+	medical_histories_id INT REFERENCES medical_histories(id)
+);
```

Comment:
- [ ] Ouch! this is close but you should also add indexes for the foreign keys found in the join table highlighted above as they're also considered foreign keys. 😉